.gitignore

```diff
@@ -8,6 +8,8 @@
 # dependencies
 node_modules
 build
+# dummy fix for https://github.com/nrwl/nx/issues/20309 and https://github.com/nx-dotnet/nx-dotnet/issues/893
+!packages/core/src/executors/build
```

Comment:
That's one funny fix! Taking care of it in the Nx TSC executor did not seem straightforward. I hope you are okay with that one.
We could also patch Nx?